package.json

```diff
@@ -4,7 +4,7 @@
     "type": "git",
     "url": "https://github.com/ArrowSphere/nodejs-api-client.git"
   },
-  "version": "3.55.0",
+  "version": "3.56.0-rc-1.nfo",
```

Comment:
If you want to publish a minor (without breaking changes) upgrade the middle number -> 3.56.0
If you want to publish a major (with breaking changes) upgrade the left number -> 4.0.0

You are not supposed to use the **rc** suffix on the main branch because it brings to official releases